### PR TITLE
docs: tells users where they are

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description
tells users where they are in sidebar
- How to fix?
added a label of Apisix ingress controller to sidebar 
___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch
https://github.com/iamayushdas/apisix-ingress-controller/tree/master

- Related commits and pull requests
https://github.com/apache/apisix-website/issues/230

- Target branch
https://github.com/apache/apisix-ingress-controller/tree/master

### CODE REVIEW

**Comment by tokers** [comment]
@juzhiyuan Please help me to review this PR, thanks! :).

File: docs/en/latest/config.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
   "version": "0.3.0",
   "sidebar": [
+    {
+      "type": "link",
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:5]: aha, a good way 😂

File: docs/en/latest/config.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+      "label": "Apisix Ingress Controller",
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:6]:
```suggestion
      "label": "Apache APISIX Ingress Controller",
```

File: docs/en/latest/deployments/ack.md [hunk 3]
```diff
@@ -39,7 +39,6 @@ This document explains how to install Ingress APISIX on [ali-cloud ACK](https://
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
 helm repo add apisix https://charts.apiseven.com
-# Use `hem search repo apisix` to search charts about apisix
```
Comment (juzhiyuan) [inline docs/en/latest/deployments/ack.md:42]: Why we need to remove this?

Comment (iamayushdas) [reply docs/en/latest/deployments/ack.md:42]: i didn't touched this file.

Comment (iamayushdas) [reply docs/en/latest/deployments/ack.md:42]: but, there's a typo in master repo in helm so i fixed that now. :)

**Comment by iamayushdas** [comment]
@juzhiyuan can i do the same for other routes too?

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
> @juzhiyuan can i do the same for other routes too?

other routes?

**Comment by iamayushdas** [comment]
> > @juzhiyuan can i do the same for other routes too?
> 
> other routes?

i mean for dashboard, apisix

**Review by gxthrj — APPROVED** [review]

**Comment by juzhiyuan** [comment]
@iamayushdas Yes

**Pull request merged**